File: C/Lista_01(Estrutura_Sequencial)/Medidas.c
#include <stdio.h>
#include <math.h>

int main () {

    double a, b, c;

    printf("Digite a medida A: ");
    scanf("%lf", &a);

    printf("Digite a medida B: ");
    scanf("%lf", &b);

    printf("Digite a medida C: ");
    scanf("%lf", &c);

    double quadrado = pow(a,2);
    double triangulo = (a*b)/2;
    double trapezio = ((a+b)*c)/2;

    printf("AREA DO QUADRADO = %.4lf\n", quadrado);
    printf("AREA DO TRIANGULO = %.4lf\n", triangulo);
    printf("AREA DO TRAPEZIO = %.4lf", trapezio);

}

File: Java/Lista_01(Estrutura_Sequencial)/Medidas.java
import java.util.Locale;
import java.util.Scanner;

public class Medidas {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner s = new Scanner(System.in);
		
		System.out.print("Digite a medida A: ");
		double a = s.nextDouble();
		
		System.out.print("Digite a medida B: ");
		double b = s.nextDouble();
		
		System.out.print("Digite a medida C: ");
		double c = s.nextDouble();
		
		double areaQ = Math.pow(a, 2);
		double areaTri = a*b/2;
		double areaTra = ((a+b)*c)/2;
		
		System.out.printf("AREA DO QUADRADO: %.4f\n", areaQ);
		System.out.printf("AREA DO TRIANGULO: %.4f\n", areaTri);
		System.out.printf("AREA DO TRAPEZIO: %.4f", areaTra);
		
		s.close();
		
	}

}
